extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
@hakubo I think @macbre did the work on Cache standarization actually :)
Anyway I think WikiaResponse::CACHE_VERY_SHORT (5 min) might be used instead of 15 minute cache time 

This is the ticket about it: https://wikia-inc.atlassian.net/browse/PLATFORM-57
I don't remember rationale clearly now, but I do remember that we used have  bazilion different cache expiration  times and it was quite  hard to divine  what expiries when, especially coupled with memcache expiration periods. 

So I think if its possible you could use the standard CACHE_VERY_SHORT.